modules/ROOT/pages/_partials/app-store-script.adoc

```diff
@@ -0,0 +1,15 @@
+=== Submission to the App Store
+
+The following does not apply if you have installed Couchbase Lite with Carthage or Cocoapods. With those package managers, Couchbase Lite is built with the application itself according to the architecture type of the target (emulator or device).
```

Comment:
I don't think that that would be the case for Carthage because you pull and build the binaries separate from the project . You should probably link folks to https://github.com/Carthage/Carthage/blob/5fd867c4895b4f59d70181dec169a1644f4430e3/README.md#if-youre-building-for-ios

Reply:
Oh I see! carthage users will have to run `carthage copy-frameworks`. Ok will update.